README.md

```diff
@@ -32,6 +32,24 @@ It is strongly recommended to upgrade your PHP version and/or install the missin
 extensions whenever possible. This polyfill should be used only when there is no
 better choice or when portability is a requirement.
 
+Usage
+=====
+
+When using [composer](https://getcomposer.org/) to manage your dependencies, you
+should not `require` the ~~`symfony/polyfill`~~ package, but the standalone ones:
```

Comment:
Cool for me ;) Thanks!
I remember that this suggestion was already mentioned in some issue/pr